package.json

```diff
@@ -49,23 +49,23 @@
     "readable-stream": "^3.6.0",
     "run-parallel-settled": "^1.0.1",
     "server-destroy": "^1.0.1",
-    "shell-quote": "^1.7.0",
+    "shell-quote": "^1.7.2",
     "tap-completed": "^1.0.0",
     "thunky-with-args": "^1.0.0",
     "transient-error": "^1.0.0",
-    "uuid": "^8.3.0",
+    "uuid": "^8.3.2",
     "watchify": "^4.0.0"
   },
   "devDependencies": {
     "abstract-browser": "^1.0.0",
-    "cross-env": "^7.0.2",
-    "depcheck": "^1.2.0",
-    "hallmark": "^3.0.0",
+    "cross-env": "^7.0.3",
+    "depcheck": "^1.4.2",
+    "hallmark": "^3.1.0",
     "nyc": "^15.1.0",
     "simple-get": "^4.0.0",
     "standard": "^14.3.4",
```

Comment:
There is a major update for this, but as this poses no security threat whatsoever, I let it be.